bin/phpcs-diff.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+DIFF_FILE=$(mktemp)
+PHPCS_FILE=$(mktemp)
+
+git remote set-branches --add origin master
+git fetch
+git diff origin/master... > $DIFF_FILE
+
+$DIR/../vendor/bin/phpcs --standard=phpcs.xml.dist --report=json > $PHPCS_FILE || true 
+
+$DIR/../vendor/bin/diffFilter --phpcs $DIFF_FILE $PHPCS_FILE 0
```

Comment:
419c4f says this:

> Settin[g] coverage to 0 so the Travis build doesn't fail on PHPCS issues

Shouldn't we get Travis to fail to force us and catch and fix the errors? Otherwise we'll just ignore them :)

Or is there some nuance about how diffFilter failures trigger?

Reply:
Ha, yea, not sure what kind of false positives it might kick up (haven't used diffFilter before), so was gonna hang off on making it a hard stop until we get a look at what it produces

Reply:
Cool, makes sense.